### DISCUSSION TITLE
key? alias to has_key? in pg_json_opts

### DISCUSSION THREAD

**Discussion by opya** [jeremyevans/sequel#2152] [General]
Hi,

I was wondering if it's wrong if we add one more alias `key?` here:  https://github.com/jeremyevans/sequel/blob/2dd31a7cf2e6f1a5d234283a4342cf8f45bc1fbf/lib/sequel/extensions/pg_json_ops.rb#L475

**Comment by jeremyevans** [comment]
I generally am against unnecessary aliases. `include?` is common enough that I made an exception for it.  I personally don't like `key?` as a method name as to me in reads like "is the receiver a key" instead of "does the receiver have the given key".  It's easy to add the alias yourself, if you want.

**Comment by opya** [comment]
ok, thank you
